Picker/CustomCellView.swift

```diff
@@ -10,19 +10,19 @@ import UIKit
 
 class CustomCellView: UICollectionViewCell {
     
-//    static let cellIdentifier = "CustomCellIdentifier"
-//    
-//    @IBOutlet weak var imageView: UIImageView!
-//    @IBOutlet weak var descriptionLabel: UILabel!
-//    @IBOutlet weak var productTypeLabel: UILabel!
-//    @IBOutlet weak var regionLabel: UILabel!
-//    @IBOutlet weak var progressView: UIProgressView!
-//    @IBOutlet weak var itemsRemaining: UILabel!
-//    @IBOutlet weak var itemsRemainingText: UILabel!
-//    @IBOutlet weak var totalAmount: UILabel!
-//    @IBOutlet weak var plegdedFor: UILabel!
-//    @IBOutlet weak var remainingHours: UILabel!
-//    @IBOutlet weak var hoursToGo: UILabel!
+    static let cellIdentifier = "CustomCellIdentifier"
+    
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var productTypeLabel: UILabel!
+    @IBOutlet weak var regionLabel: UILabel!
+    @IBOutlet weak var progressView: UIProgressView!
+    @IBOutlet weak var itemsRemaining: UILabel!
```

Comment:
We essentially have to keep the names of the labels uniform. So if you're making `descriptionLabel` we should have `itemsRemainingLabel` too